models/JobSeekerModel.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require("mongoose");
+const { constants } = require("../helpers/constants");
+const SkillSchema = require("./SkillModel");
+const Schema = mongoose.Schema;
+const DataTypes = Schema.Types;
+
+const JobSeekerSchema = new Schema(
+  {
+    experienceType: {
+      type: String,
+      enum: Object.values(constants.experienceType),
+      default: constants.experienceType.FRESHER,
+      required: true,
+    },
+    experience: {
+      type: DataTypes.Number,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 3 tabs but found 6 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
			type: DataTypes.Number,
```